packages/protocol/contracts/standards/IERC900.sol

```diff
@@ -52,4 +52,8 @@ interface ERC900 {
      * @return True if the optional history functions are implemented, false otherwise
      */
     function supportsHistory() external pure returns (bool);
+
+    function lastStakedFor(address addr) external view returns (uint256);
```

Comment:
To be consistent, let's add the natspec and use the `_` prefix for the params :)